### PR TITLE
sqlite: update to 3.34.1

### PR DESCRIPTION
**Pull request by heitbaum**
updtate 3.34.0 to 3.34.1
changelog: https://www.sqlite.org/releaselog/3_34_1.html

### CODE REVIEW

File: packages/databases/sqlite/package.mk [hunk 1]
```diff
@@ -3,11 +3,11 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="sqlite"
-PKG_VERSION="3.34.0"
-PKG_SHA256="bf6db7fae37d51754737747aaaf413b4d6b3b5fbacd52bdb2d0d6e5b2edd9aee"
+PKG_VERSION="3.34.01"
```
Comment (CvH) [inline packages/databases/sqlite/package.mk:6]: its 3.34.1

Comment (heitbaum) [reply packages/databases/sqlite/package.mk:6]: Yes, but unhappily they have named the file 3340100 and our bash magic fails.

Comment (CvH) [reply packages/databases/sqlite/package.mk:6]: there is no easy way to do it :D
maybe this way ?
```
PKG_VERSION="3.34.1"
PKG_VERSION_SQLITE="${PKG_VERSION/./}00"
PKG_URL="https://www.sqlite.org/2021/${PKG_NAME}-autoconf-${PKG_VERSION_SQLITE/./0}.tar.gz"
```

Comment (SupervisedThinking) [reply packages/databases/sqlite/package.mk:6]: what about https://github.com/sqlite/sqlite/releases/tag/version-3.34.1 🤔

Comment (CvH) [reply packages/databases/sqlite/package.mk:6]: looks very different content wise, the tea directory does not even exist

Comment (heitbaum) [reply packages/databases/sqlite/package.mk]: Done.

**Review by CvH — APPROVED** [review]

**Pull request merged**